assets/xml/objects/object_famos.xml

```diff
@@ -24,7 +24,7 @@
         <Limb Name="gFamosHeadLimb" Type="Standard" EnumName="FAMOS_LIMB_HEAD" Offset="0x3D18" />
         <Skeleton Name="gFamosSkeleton" Type="Normal" LimbType="Standard" LimbNone="FAMOS_LIMB_NONE" LimbMax="FAMOS_LIMB_MAX" EnumName="FamosLimb" Offset="0x3D38" />
 
-        <Animation Name="gFamosIdleAnim" Offset="0x3DC8" />
+        <Animation Name="gFamosIdleAnim" Offset="0x3DC8" /> <!-- Original name is "fam_dosun" -->
```

Comment:
Er, any idea what this is? Japanese onomatopoeia for flying maybe?

Reply:
Apparently, it's onomatopoeic for "bam; whomp; thump". Kind of don't see how this really applies to an idle animation, but oh well.